packages/as-types/assembly/uint8ArrayToBase64.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Converts a Uint8Array to a base64 string.
+ *
+ * @param bytes - The Uint8Array to convert.
+ *
+ * @returns The base64 string.
+ */
+export function uint8ArrayToBase64(bytes: Uint8Array): string {
```

Comment:
why not using https://github.com/near/as-base64 instead ?

Reply:
I didn't know about this library